peekingduck/pipeline/nodes/model/hrnet.py

```diff
@@ -24,9 +24,59 @@
 
 class Node(AbstractNode):
     """HRNet node class that initialises and use hrnet model to infer poses
-    from detected bboxes
+    from detected bboxes.
 
-    Deep High-Resolution Representation Learning for Visual Recognition
+    The HRNet is a universal architecture for visual recognition. The
+    HRNet has become a standard for human pose estimation since the paper
+    was published in CVPR 2019. The HRNet applied to human pose estimation
+    uses the representation head, called HRNetV1. HRNet outperforms ResNet
+    in terms of estimation performance (AP), parameter complexity
+    (#parameters), and computation complexity (GFLOPS).
```

Comment:
this one here is not really necessary. possible to shorten it to 1 or 2 lines?

Reply:
Done